### DISCUSSION TITLE
Using disk-utilities to validate disk images.

### DISCUSSION THREAD

**Discussion by dwbridgfoot** [keirf/disk-utilities#195] [General]
In greasweazle discussion # 167 it was mentioned that:
"Best in my opinion for archive is a flux dump that has been validated using an analyser like disk-utilities for example."
I'm wondering what the disk-utilities command would look like to validate an Amiga SCP disk image?
Thanks.

**Comment by keirf** [comment]
`disk-analyse --format=amigados name.scp name.ipf`

The good thing is disk analyse knows lots of custom track formats and copy protections on Amiga, and has a database of games. So format here could be "New Zealand Story" for example, and the Copylock protection would be correctly validated.

**Reply by dwbridgfoot** [reply]
Awesome, thanks for the quick reply. 
I'm not really sure what to expect but this is the result of one example:
T0.0: AmigaDOS
T0.1: Unformatted
T1.0-79.1: AmigaDOS
T80.0-83.1: Unformatted

Is it just a case of making sure there are no issues/errors?
& is it worth keeping the resultant .IPF file as opposed to the .SCP version? It's definitely a hell of a lot smaller size compared to a 5 pass .SCP.

**Reply by keirf** [reply]
I'd be skeptical that T0.1 is really empty. It's often a protection track. That's why keeping the scp is useful, and local storage is cheap these days.

Perhaps try format `amigados_copylock`

**Reply by dwbridgfoot** [reply]
Hmm, same result. 
But that's okay, I don't mind having the larger .SCP. I was mainly just looking for a way to check or verify that the image I've made is 'good'.
I'll keep playing & try some different images.
